frontend/src/Commons/Forms/InputModules/Crops/CropsInfo.styles.ts

```diff
@@ -4,9 +4,9 @@ import * as tokens from '@bcgov/design-tokens/js';
 
 const StyledCropsSmallGroup = styled.div`
   display: flex;
-  gap: 20px;
+  gap: 24px;
```

Comment:
This is the spacing for vertical, horizontal might change. But vertical alignment is important to be 24px at all times.